website/templates/project/project_header.mako

```diff
@@ -78,6 +83,22 @@
     </style>
 % endif
 
+% if node['is_registration'] and node['pending_retraction']:
+    <div class="alert alert-info">This ${node['node_type']} is currently pending entering into a retracted state.</div>
+% endif
+
+% if node['is_registration'] and node['is_retracted']:
```

Comment:
Rather than having multiple checks for `node['is_registration'] , it would be more concise to have the additional checks for `is_retracted` , etc. in nested ifs.